### ISSUE TITLE
WoW errors

### ISSUE THREAD

**Issue by pablobit94** [WowDevTools/CASCHost#26]
Hey. I've one WoW error and i need repair it.
I'm using Ubuntu and CascHost in Static mode. 7.3.5 26972 Version.

When I use my static CDN Blizzlike files, i don't have wow error and all right. But if I put somes custom files, i've wow error somes times if I put much custom files, if I put a little custom files, i've less wow errors.

The wow error is this:
[20190712T12:55:31] {10dc} INF: NGDP initialization - (archive: true, cache: true, Async: true)
[20190712T12:55:31] {3b90} INF: adding network address 188.165.192.135:80 for server //cdn.cenizasdeazeroth.com
[20190712T12:55:31] {3b90} INF: adding network address 188.165.192.135:80 for server //wow.cenizasdeazeroth.com
[20190712T12:55:31] {3b90} INF: adding network address 188.165.192.135:80 for server //cda.cenizasdeazeroth.com
[20190712T12:55:31] {3b90} INF: adding network address 188.165.192.135:80 for server //logon.cenizasdeazeroth.com
[20190712T12:55:31] {3b90} INF: adding network address 188.165.192.135:80 for server //legion.cenizasdeazeroth.com
[20190712T12:55:31] {3b90} INF: adding network address 188.165.192.135:80 for server //server.cenizasdeazeroth.com
[20190712T12:55:31] {3b90} INF: adding network address 67.26.101.254:80 for server //level3.blizzard.com
[20190712T12:55:31] {3b90} INF: adding network address 8.247.221.126:80 for server //level3.blizzard.com
[20190712T12:55:31] {3b90} INF: adding network address 8.241.107.126:80 for server //level3.blizzard.com
[20190712T12:55:31] {3b90} INF: adding network address 67.26.99.254:80 for server //level3.blizzard.com
[20190712T12:55:31] {3b90} INF: adding network address 8.247.215.254:80 for server //level3.blizzard.com
[20190712T12:55:31] {3b90} INF: NameResolutionComplete numServers 7 numHosts 11
[20190712T12:55:32] {10dc} INF: Initialization step - FETCHING_CDN_CONFIG
[20190712T12:55:32] {10dc} INF: Initialization step - FETCHING_BUILD_CONFIG
[20190712T12:55:32] {10dc} INF: no patch configuration
[20190712T12:55:32] {10dc} ERR: empty group key
[20190712T12:55:32] {10dc} INF: failed to load group index '', re-merging indices
[20190712T12:55:52] {3b90} WRN: fetcher content lag, data rejected
[20190712T12:55:53] {3b90} WRN: fetcher content lag, data rejected
[20190712T12:55:54] {3b90} WRN: fetcher content lag, data rejected
[20190712T12:55:54] {3b90} WRN: fetcher content lag, data rejected
[20190712T12:55:56] {3b90} WRN: fetcher content lag, data rejected

**Steps to reproduce:**
- Unrar CDN Blizzlike files in my apache folder home/www/tpr and /home/www/wow
- Insert custom files in my CascHost /wwwroot/Data folder.
- Get this [.build.info](https://drive.google.com/file/d/1kAzKbwHaVq76BkWHRHjNVZODHqTOvM6G/view?usp=sharing) modified by me and insert these file to /wwwroot/SystemFiles
- Insert somes files config in SystemFiles: 
![image](https://user-images.githubusercontent.com/35303073/61124035-28f23880-a4a6-11e9-9f8d-15d185b3c4a2.png)
- Config appsettings.json with esES, and database connection config and my cdn domain
- I did "dotnet run" in CascHost.
- I get the Output folder and copy this files to /home/www/tpr/wow/ and the versions file to /home/www/wow/ and overwrite these file only.
- I modified the cdns files
- Now I have this two files [CDNs](https://drive.google.com/file/d/1f6FTKrGHL4fYXScZ1KrzVZD7MrmWx1j9/view?usp=sharing) and [VERSIONS](https://drive.google.com/file/d/1Z2y7gFMJWlJB0EEE3GOeGsBufo0hkdCY/view?usp=sharing)
- I open my Wow.exe modified with my cdn config.
- Somes times it is work, but other times give me Wow error.

**Expected Behaviour:**
I want open my Wow without wow errors.

**Actual Behaviour:**
I obtain wow error when i load one map and I need reload my WoW somes times for this work.

**Comment by MaxtorCoder** [comment]
You know that CASCHost has a built in webserver you can use? Instead of the standard XAMPP server?

**Comment by pablobit94** [comment]
Yes, but i had somes problems and i was using apache2. Do you think if i use webserver of Caschost, it is work better?

And how can I use my CDN Blizzlike files with webserver of Caschost?

**Comment by MaxtorCoder** [comment]
Just put them in the output folder lol

**Comment by pablobit94** [comment]
Hmm and i must put Caschost in dynamic mode or static mode for i can do it?
Thanks
